impeller/renderer/backend/vulkan/texture_vk.h

```diff
@@ -6,18 +6,31 @@
 
 #include "flutter/fml/macros.h"
 #include "impeller/base/backend_cast.h"
+#include "impeller/renderer/backend/vulkan/context_vk.h"
+#include "impeller/renderer/backend/vulkan/vk.h"
 #include "impeller/renderer/texture.h"
 
 namespace impeller {
 
 class TextureVK final : public Texture, public BackendCast<TextureVK, Texture> {
  public:
-  TextureVK(TextureDescriptor desc);
+  TextureVK(TextureDescriptor desc,
+            ContextVK& context,
+            const VmaAllocator& allocator,
+            VkImage image,
+            VmaAllocation allocation,
+            VmaAllocationInfo allocation_info);
 
   // |Texture|
   ~TextureVK() override;
 
  private:
+  ContextVK& context_;
+  const VmaAllocator& allocator_;
+  VkImage image_;
```

Comment:
Shouldn't these be unique images?

Reply:
Also, zero out these variable explicitly in case we add more ctors.

Reply:
I thought the image was just a "holder" to the vma managed allocation, and in the dtor i clean it up. Should we still have a `UniqueImage`?